com.coala.core/plugin.xml

```diff
@@ -57,7 +57,7 @@
       <menuContribution
             locationURI="popup:org.eclipse.ui.popup.any?after=additions">
          <menu
-               icon="icons/coala-analyzer.gif"
+               icon="icons/coala.gif"
```

Comment:
@thotayashwanth123 changed and committed the changes? still its failing..I cannot see your rename commit!! can you plz do it so i can see trace back!!